bundle/language_en.properties

```diff
@@ -187,6 +185,11 @@ angal.lab.newlaboratoryexam.title
 angal.lab.editlaboratoryexam.title                                               = Edit Laboratory Exam 
 angal.login.title                                                                = Login
 angal.mainmenu.fmt.title                                                         = User: {0}
+angal.messagedialog.error.title                                                  = Error
+angal.messagedialog.warning.title                                                = Warning
+angal.messagedialog.info.title                                                   = Information
+angal.messagedialog.question.title                                               = Question
+angal.messagedialog.note.title                                                   = Note
```

Comment:
if you wish, we can put `angal.messagedialog.plain.title` as you was proposing, for consistency with JOptionPane

Reply:
Done